app/src/sharedTest/java/org/oppia/app/settings/profile/ProfileListFragmentTest.kt

```diff
@@ -45,7 +45,7 @@ import javax.inject.Singleton
 
 @RunWith(AndroidJUnit4::class)
 @LooperMode(LooperMode.Mode.PAUSED)
-class ProfileListActivityTest {
+class ProfileListFragmentTest {
```

Comment:
Test  cases in this file still use `testProfileListActivity`, update that too.

Reply:
Done